integration/terraform/ec2/linux/README.md

```diff
@@ -10,3 +10,15 @@ Instance assumptions
 6. make
 7. aws-cli
 8. CloudWatchAgentServerRole is attached
+9. crontab
+
+**How are ami built?**
+
+AMI builder pipeline builds the ami
+
+The pipeline installs required packages and updates ami software
+
+This process generates a new ami we can then use for testing
+
+Required packages https://github.com/aws/amazon-cloudwatch-agent/blob/master/integration/terraform/ec2/linux/README.md
```

Comment:
Kind of confusing to have this link to itself. IIRC you can use link anchors in Markdown, but don't recall how.